Assets/Scripts/PlayerScript.cs

```diff
@@ -4,20 +4,21 @@
 
 public class PlayerScript : MonoBehaviour
 {
-    //Scripts
+    [Header("Scripts")]
     public InputManager inputManager;
     public PlayerMovement playerMovement;
     public PlayerCollision playerCollision;
+    [SerializeField] private ZeeTimeSlow zeeTimeSlow = null;
 
-    //Components
+    [Header("Components")]
     [HideInInspector] public Rigidbody2D rb;
     [HideInInspector] public SpriteRenderer spriteRenderer;
     private Animator anim;
 
-    //Player attributes
+    [Header("Player Attributes")]
```

Comment:
Great idea to add Headers! Really cleans up the Inspector 👍🎉🎉

Reply:
YEAH SUPER GREAT IDEA